src/client.cpp

```diff
@@ -527,8 +527,8 @@ ProcessResponseEnum HttpClient::ProcessResponse(Value& result, bool notification
 bool TcpClient::GenerateHeader()
 {
     log_trace();
-    header_ << request_.Length() << ":";
-    log_debug("Request length=" << request_.Length());
+    header_ << static_cast<uint64_t>(request_.Length()) << ":";
```

Comment:
Alternatively we could do, what the STL impl does:

```
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, char c);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, const char* str);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, const std::string& str);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, int i);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, unsigned int u);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, long int u);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, unsigned long int u);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, long long int u);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, unsigned long long int u);
ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, double d);
```

One implementation would look like this (they are all the same):
```
anyrpc::Stream& operator<<(anyrpc::Stream& os, unsigned int u)
{
    os.Put(std::to_string(u));
    return os;
}
```

That way we cover all the integer types there are 
see: http://en.cppreference.com/w/cpp/language/types